src/renderer/shared/api/translation/locales/en.json

```diff
@@ -378,11 +380,46 @@
         "votes": "Delegated votes"
       },
       "title": "Add delegation on",
+      "tracks": {
+        "crowdloans": "Auction admin",
+        "crowdloansDescription": "Origin for starting auctions.",
+        "fellowshipAdmin": "Fellowship: Admin",
+        "fellowshipAdminDescription": "Origin for managing the composition of the fellowship.",
+        "fellowshipWhitelist": "Fellowship: Whitelist",
+        "fellowshipWhitelistDescription": "Origin commanded by members of the Polkadot Fellowship.",
+        "governanceCanceller": "Governance: Canceller",
+        "governanceCancellerDescription": "Origin able to cancel referenda.",
+        "governanceKiller": "Governance: Killer",
+        "governanceKillerDescription": "Origin able to kill referenda.",
+        "governanceLease": "Lease admin",
+        "governanceLeaseDescription": "Origin able to force slot leases.",
+        "governanceRegistrar": "Governance: Registrar",
+        "governanceRegistrarDescription": "Origin for managing the registrar.",
+        "mainAgenda": "Root",
```

Comment:
Can you check it with the designers, I'm not sure that we should have 
different names  for the same track like 'Root' and 'Main agenda'